plugin-interface/src/main/scala/mesosphere/marathon/plugin/EnvVarValue.scala

```diff
@@ -0,0 +1,11 @@
+package mesosphere.marathon.plugin
+
+trait EnvVarValue
+
+trait EnvVarString extends EnvVarValue {
```

Comment:
Why do we need these types here anyway? Also case classes can be extended if some plugin needs to.